data/yy

```diff
@@ -0,0 +1,4 @@
+include:duowan
+
+yy.com
```

Comment:
1. gif tracker from `https://mlog.hiido.com/j.gif` and `https://ylog.hiido.com/j.gif`. consider mark `mlog.hiido.com` and  `ylog.hiido.com` as tracker.
2. same tracker with hardcoded address: `https://14.17.112.234/j.gif`. consider mark `14.17.112.234` as tracker.

Reply:
updates: code found in `https://web.yystatic.com/project/yycom/pc/js/ventors-18030b476d.js`:
```javascript
domain: 'ylog.hiido.com',
        ipPrefix: '14.17.112.',
        ips: [
          232,
          233,
          234,
          235,
          236,
          237
        ],
```
this code is followed by a random selection. 
I suggest add these addresses as tracker.

Reply:
IP addresses should be handled by GeoIP or content blocker instead of this project. So there's not much we can do here.

Reply:
umm... sad.